test/blackbox-tests/test-cases/watching/dune

```diff
@@ -17,8 +17,9 @@
  (applies_to what-dune-watches)
  (deps %{bin:strace}))
 
-;; this test sometimes gets stuck and times out
+;; These tests sometimes get stuck and time out:
 
 (cram
- (applies_to watching-eager-concurrent-build-command)
+ (applies_to watching-eager-concurrent-build-command
+  watching-eager-concurrent-exec-command)
```

Comment:
But doesn't that mean that the tests are basically never run? Not on CI, not on people's machines when they `dune runtest`?

Reply:
I agree it's not ideal but I don't have a reliable way to reproduce the problem. The tests are reliable on my dev machines and only fail from time to time in CI. Here's the PR that disabled this test initially: https://github.com/ocaml/dune/pull/11850. There are some other watch-mode tests that are disabled for similar reasons.